gatsby_fp/src/components/Layout/Body/Fields/IntellCont/SubHeading.js

```diff
@@ -32,128 +32,97 @@ export default function IntellCont({ intellContList, publicationType }) {
     return sortedIntellContList
   }
 
-    const sortedIntellContList = sortStatusAndYear(intellContList)
+  const sortedIntellContList = sortStatusAndYear(intellContList)
+  const liInnerHtmlList =
+    sortedIntellContList.filter(element => (element.contype === publicationType || element.contypeother === publicationType)).map(element => {
+      let liString = '';
+      if (element.status) {
+        liString += `${element.status} `
+      }
+      if (element.contype && !element.contypeother) {
+        liString += `${element.contype} - `
+      }
+      if (element.contypeother) {
+        liString += `${element.contypeother} - `
+      }
+      element.intellcont_auth.forEach(elem => {
+        if (elem.lname) {
+          liString += `${elem.lname}, `
+        }
+        if (elem.fname && !elem.mname) {
+          liString += `${elem.fname.substr(0,1)}.; `
```

Comment:
This semicolon-space delimiter is appearing after every author. It should only appear between authors.

This also applies to the next few lines, which have the same issue.

You could use the second parameter of `.forEach` to get an index, and conditionally add the delimiter only when the index is less than `element.intellcont_auth.length - 1`. Conditionally adding the delimiter can exist just once, no need to do it 3 times, by putting it at the very end of the `forEach` after all the existing conditionals.